### PR TITLE
feat(Alert): create Alert component

### PR DESCRIPTION
**Pull request by spartDev**
- [x] Feature
- [ ] Fix
- [ ] Enhancement

## Brief
Create Alert component

## Description
Create Alert component

## Todo - Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.

### CODE REVIEW

File: src/Alert/Container.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  };
+
+  /**
+   * Component internal state
+   */
+  state = {
+    status: INACTIVE,
+  };
+
+  componentDidMount() {
```
Comment (AntoninMarchardDev) [inline src/Alert/Container.js]: Add documentation for each methods

File: src/Alert/Container.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  }
+
+  /**
+   * Start Timer
+   *
+   */
+  startTimer = () => {
+    const { timeout } = this.props;
+
+    this.timer = setTimeout(this.hide, timeout);
```
Comment (AntoninMarchardDev) [inline src/Alert/Container.js]: Why not putting the `timer` as a component state ?

File: src/Alert/Provider.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+   * Hide
+   *
+   * Hide the alert container
+   *
+   */
+  hide = () => {
+    this.setState({ alert: {}, active: false });
+  };
+
+  render() {
```
Comment (AntoninMarchardDev) [inline src/Alert/Provider.js]: Doc

File: src/Message/index.js [hunk 4]
```diff
@@ -21,42 +23,101 @@ import { getStatusBackgroundColor } from './helpers';
...
+   */
+  static defaultProps = {
+    ariaLabel: null,
+    className: '',
+    onClose: null,
+    onCloseText: '',
+    type: 'info',
+  };
+
+  componentDidMount() {
```
Comment (AntoninMarchardDev) [inline src/Message/index.js]: doc

File: src/Message/index.js [hunk 5]
```diff
@@ -21,42 +23,101 @@ import { getStatusBackgroundColor } from './helpers';
...
+    const elButton = this.closeButtonRef.current;
```
Comment (AntoninMarchardDev) [inline src/Message/index.js]: Find another naming for this, `elButton` is not satisfying

File: src/helpers/objectIsEmpty.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+ * isEmpty
+ *
+ * Check if an object is empty
+ *
+ * @param {object} obj
+ *
+ * @return {boolean}
+ */
+
+function isEmpty(obj) {
```
Comment (AntoninMarchardDev) [inline src/helpers/objectIsEmpty.js]:
```js
const isObjectEmpty => {}
```

**Comment by AntoninMarchardDev** [comment]
@spartDev fix conflicts

**Comment by spartDev** [comment]
done!

**Pull request closed without merging**